src/url_util.ts

```diff
@@ -363,9 +363,20 @@ export function interpolateSearchItem(urlPattern: URL, query: string): URL {
         query = encodeURIComponent(query)
     }
 
+    queryWords = query.split(" ")
```

Comment:
Did you test this? Travis says that this hasn't been defined amongst other things. I think you're probably missing a few lets.

Reply:
Fixed by @glacambre